pkg/nsx/services/vpc/vpc_test.go

```diff
@@ -37,16 +37,17 @@ import (
 )
 
 var (
-	vpcName1          = "ns1-vpc-1"
-	vpcName2          = "ns1-vpc-2"
-	infraVPCName      = "infra-vpc"
-	vpcID1            = "ns-vpc-uid-1"
-	vpcID2            = "ns-vpc-uid-2"
-	vpcID3            = "ns-vpc-uid-3"
-	IPv4Type          = "IPv4"
-	cluster           = "k8scl-one"
-	tagScopeCluster   = common.TagScopeCluster
-	tagScopeNamespace = common.TagScopeNamespace
+	vpcName1             = "ns1-vpc-1"
+	vpcName2             = "ns1-vpc-2"
+	infraVPCName         = "infra-vpc"
+	vpcID1               = "ns-vpc-uid-1"
+	vpcID2               = "ns-vpc-uid-2"
+	vpcID3               = "ns-vpc-uid-3"
+	IPv4Type             = "IPv4"
+	cluster              = "k8scl-one"
+	tagScopeCluster      = common.TagScopeCluster
+	tagScopeNamespace    = common.TagScopeNamespace
+	tagScopeNamespaceUID = common.TagScopeNamespaceUID
```

Comment:
There is an error at code line#L63
cache.Indexers{common.TagScopeStaticRouteCRUID: indexFunc}),
```
vpcStore := &VPCStore{ResourceStore: common.ResourceStore{
		Indexer:     cache.NewIndexer(keyFunc, cache.Indexers{common.TagScopeStaticRouteCRUID: indexFunc}),
		BindingType: model.VpcBindingType(),
```
}}

Reply:
line63? Could you make a more clear code suggestion?